stm32-modules/include/thermocycler-gen2/thermocycler-gen2/motor_task.hpp

```diff
@@ -1289,8 +1289,10 @@ class MotorTask {
                     motor_util::LidStepper::Position::CLOSED;
                 // The overall lid state machine can advance now
                 error = handle_lid_state_end(policy);
-                // TODO(Frank, Mar-7-2022) check if the lid didn't make it in
-                // all the way
+                // if the lid isn't actually closed, overwrite error status
```

Comment:
Do we want to do this before the `handle_lid_state_end` call above?

Reply:
I figured it would be easier to overwrite the error value after that function is done, since inside of its scope it doesn't know the desired status of the lid, which in this case is `OVERDRIVE`